dll/cpl/desk/lang/tr-TR.rc

```diff
@@ -154,79 +154,79 @@ END
 
 IDD_ADVANCED_GENERAL DIALOGEX 0, 0, 253, 204
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Umûmî"
+CAPTION "Genel"
 FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Görüntü", -1, 7, 6, 237, 56, WS_GROUP
     LTEXT "Yazı Tipi Boyutu", -1, 14, 20, 222, 9
     COMBOBOX IDC_FONTSIZE_COMBO, 14, 30, 223, 80, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     LTEXT "", IDC_FONTSIZE_CUSTOM, 14, 46, 223, 12
     GROUPBOX "Uyumluluk", -1, 7, 67, 237, 97
-    LTEXT "Eğer renk ayarlarını değiştirdiğinizden sonra bilgisayarınızı yeniden başlatmazsanız birtakım izlenceler yanlış çalışır.\r\n\nRenk ayarlarını değiştirdikten sonra:", -1, 12, 78, 228, 34
-    AUTORADIOBUTTON "&yeni renk ayarları uygulanmadan önce bilgisayarı yeniden başlat.", IDC_RESTART_RB, 14, 116, 228, 10
-    AUTORADIOBUTTON "y&eniden başlatmadan yeni renk ayarlarını uygula.", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10
-    AUTORADIOBUTTON "ye&ni renk ayarlarıuygulanmadan önce bana sor.", IDC_ASKME_RB, 14, 146, 228, 10
+    LTEXT "Eğer renk ayarlarını değiştirdiğinizden sonra bilgisayarınızı yeniden başlatmazsanız bazı programlar yanlış çalışır.\r\n\nRenk ayarlarını değiştirdikten sonra:", -1, 12, 78, 228, 34
+    AUTORADIOBUTTON "y&eni renk ayarları uygulanmadan önce bilgisayarı yeniden başlat", IDC_RESTART_RB, 14, 116, 228, 10
```

Comment:
I've checked on english & other languages and these are full sentences starting with a capital letter and ending with a full stop.

Reply:
Ok, I've updated these strings.